### PR TITLE
UPDATED: jupyterLab desktop launcher.

### PR DESCRIPTION
**Pull request by Axlfc**
Solving #133

### CODE REVIEW

File: data/features/jupyterLab/jupyterLab.dat.sh [hunk 1]
```diff
@@ -10,7 +10,10 @@ jupyterLab_binariesinstalledpaths=("bin/jupyter-lab;jupyter-lab" "bin/jupyter;ju
 jupyterLab_packagedependencies=("libkrb5-dev")
 jupyterLab_flagsoverride=";;1;;;"  # Ignore Errors to check dependencies. This is a patch
 jupyterLab_launcherkeynames="defaultLauncher"
-jupyterLab_defaultLauncher_exec="jupyter-lab &"
+jupyterLab_defaultLauncher_exec="bash ${CURRENT_INSTALLATION_FOLDER}/jupyterLab.sh"
```
Comment (AleixMT) [inline data/features/jupyterLab/jupyterLab.dat.sh:13]: I think that you should do a link jupyterLab.sh as the command jupyter and use this as the command to call in the desktop launcher.

File: data/features/jupyterLab/jupyterLab.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+jupyter-lab &
```
Comment (AleixMT) [inline data/features/jupyterLab/jupyterLab.sh:2]: Plus, the command has not got its output silenced

File: data/features/jupyterLab/jupyterLab.dat.sh (hunk 1 above)
Comment (Axlfc) [reply data/features/jupyterLab/jupyterLab.dat.sh:13]: jupyter is not the same as jupyter-lab

File: data/features/jupyterLab/jupyterLab.sh (hunk 2 above)
Comment (Axlfc) [reply data/features/jupyterLab/jupyterLab.sh:2]: A terminal is never seen when using the desktop launcher so no output.

File: data/features/jupyterLab/jupyterLab.dat.sh (hunk 1 above)
Comment (AleixMT) [reply data/features/jupyterLab/jupyterLab.dat.sh:13]: Well, the point is that you should use a link. I do not care how do you call it.

File: data/features/jupyterLab/jupyterLab.sh (hunk 2 above)
Comment (AleixMT) [reply data/features/jupyterLab/jupyterLab.sh:2]: If you use a link as I suggested, you will need to.

**Review by AleixMT — CHANGES_REQUESTED** [review]
I also added a deprecation notice in a new issue. You may solve this first and then deprecate the old kernels.

File: data/features/jupyterLab/jupyterLab.dat.sh (hunk 1 above)
Comment (AleixMT) [reply data/features/jupyterLab/jupyterLab.dat.sh:13]: @Axlfc

File: data/features/jupyterLab/jupyterLab.sh (hunk 2 above)
Comment (AleixMT) [reply data/features/jupyterLab/jupyterLab.sh:2]: @Axlfc

**Comment by AleixMT** [comment]
Can you implement the link and finish this? @Axlfc

**Comment by AleixMT** [comment]
Jupyter Notebook is broken. It needs maintainance @Axlfc to delete old kernels and create a link for the binary.  I am closing this issue since you ignored my commentaries and change requests. Do another issue for these.

**Comment by AleixMT** [comment]
Since you are not addressing the requested changes I will be deprecating jupyter lab feature

**Pull request closed without merging**